README.md

```diff
@@ -32,6 +32,19 @@ make
 
 If all runs successfully, you should have an executable.
 
+#### Linux
+[Note: 'make install' is broken at the moment. Just copy the binary 'projectMSDL' to your choice of run-path. E.g.]
```

Comment:
Not broken, but unimplemented actually, yet the implications are the same.
There are basic install commands [in my UI implementation branch](https://github.com/kblaschke/frontend-sdl2/blob/imgui-test/install.cmake), but none in master yet.